pywt/tests/test_cwt_wavelets.py

```diff
@@ -371,3 +371,51 @@ def test_cwt_small_scales():
 
     # extremely short scale factors raise a ValueError
     assert_raises(ValueError, pywt.cwt, data, scales=0.01, wavelet='mexh')
+    
+
+def test_cwt_method_fft():
```

Comment:
If you add an `import for pytest`  and define
```Python
float_types = [np.float32, np.float64, np.complex64, np.complex128]
```

You can use parametrize to repeat tests across dtypes as shown below. (We only recently switched to using `pytest` so this is not currently done elsewhere in the test suite)

```Python
@pytest.parametrize('dtype', float_types)
def test_cwt_method_fft(dtype):
    data = np.zeros(32, dtype=dtype)
```

Reply:
Altenatively, you could also use parametrize with an additional parameter for method = 'fft' or 'auto' to combine these two separate test functions into one:

 (adding `import itertools` at the top)
```Python
@pytest.mark.parametrize(
    'dtype, method', itertools.product(float_types, ['fft', 'auto'])
def test_cwt_methods(dtype, method):
    data = np.zeros(32, dtype=dtype)
```